docs/editor_agents/modal.md

```diff
@@ -48,7 +48,7 @@ Now you need a template for creating an endpoint that can be used for defining a
 The following example:  
 
 1. **Creates a Docker Image** – We define a container that includes all necessary dependencies:  
-   - `encord-agents`: The library for building agents
+   - `encord-agents[headless]`: The library for building agents
```

Comment:
As in the PR i did on top of this one, I'd prefer having explicit names `opencv` and `opencv-headless` as the optional dependencies. Goes for all the pip installs below.